crates/radix-engine-toolkit/tests/transaction_types.rs

```diff
@@ -2645,8 +2522,57 @@ fn lock_fee_manifest_has_no_classification_except_general() {
         dynamic_analysis.detailed_classification.len(),
         1
     );
-    assert_eq!(
-        static_analysis.classification,
-        indexset![ManifestClass::General]
-    );
+    assert_eq!(static_analysis.classification, indexset![
+        ManifestClass::General
+    ]);
+}
+
+#[test]
+fn subintent_manifest_of_transfer_is_a_general_subintent() {
+    let account =
+        ComponentAddress::new_or_panic([EntityType::GlobalAccount as u8; 30]);
+    subintent_manifest_classification_test(
+        ManifestBuilder::new_subintent_v2()
+            .lock_fee(account, dec!(10))
+            .withdraw_from_account(account, XRD, 10)
+            .take_all_from_worktop(XRD, "bucket")
+            .try_deposit_or_abort(account, None, "bucket")
```

Comment:
Can we have a test which includes a verify parent, as we need that to work too? :)